test/install.Tests.ps1

```diff
@@ -0,0 +1,29 @@
+BeforeAll {
+    # Load SUT
+    $sut = (Split-Path -Leaf $PSCommandPath).Replace('.Tests.ps1', '.ps1')
+    . ".\$sut"
```

Comment:
Just noticed this will actually trigger an installation of Scoop, it tries to expose functions for unit tests below but it imports the whole install script. I noticed it when implementing #76 and the ci kept failing. ;p